src/lib/openapi-to-json-schema.ts

```diff
@@ -786,6 +786,13 @@ export default function toJSONSchema(
 
   // Only add a default value if we actually have one.
   if ('default' in schema && typeof schema.default !== 'undefined') {
+    // If it's an enum, add the default to the description if it's not in the response schema.
+    if ('enum' in schema && !addEnumsToDescriptions) {
+      schema.description = schema.description
+        ? `${schema.description} default: ${schema.default}`
+        : `default: ${schema.default}`;
```

Comment:
```suggestion
        ? `${schema.description}\n\nDefault: ${schema.default}`
        : `Default: ${schema.default}`;
```

Reply:
Also not at all opposed to you trimming any trailing `\n`'s off descriptions we set in the JSON Schema because with this it'll end up adding 3 `\n`'s for those cases which'll look like crap.